exercises-latex/example-lab-report/loesung/v16516/content/auswertung.tex

```diff
@@ -0,0 +1,121 @@
+\section{Auswertung}
+\label{sec:Auswertung}
+
+
+Die für diesen Versuch relevanten physikalischen Größen für Kugel und Zylinder sind 
+
+\begin{align}
+  \label{eq:physical-properties}
+  m_\text{Z} &= \input{mass_ball.tex} & m_\text{K} &= \input{mass_ball.tex}\nonumber\\
+  r_{\text{Z},\text{i}} &= \input{../build/radius-inner_cylinder.tex} & r_\text{K} &= \input{radius_ball.tex}\\
+  r_{\text{Z},\text{a}} &= \input{../build/radius-outer_cylinder.tex} &&\nonumber
+\end{align}
+
+Berechnet wurde aus diesen Größen das jeweilige, theoretiche Trägheitsmoment
+
+\begin{align}
+  \label{eq:moments-inertia}
+  I_\text{Z} &= \input{./../build/theoretical-I_cylinder.tex} && I_\text{K} = \input{./../build/theoretical-I_ball.tex}.
+\end{align}
+
+
+
+Aus den aufgenommenen Videos der Versuche wurden die in \autoref{tab:all-measurements} folgenden Frameindizes extrahiert.
+Die ebenfalls aufgeführte Zeitdauer $t$ wurde dann jeweils aus der aus dem Startframeindex $F_\text{i}$,
+dem Endframeindex $F_\text{f}$ und der Kameraframerate $\symup{fps}=\input{framerate.tex}$ wie folgt 
```

Comment:
here I would have the framerate unit with a `per-mode=reciprocal`

Reply:
symup is the wrong command for fps, it still treats it as f • p • s, just upright.

Correct would be operatorname